test/powershell/Language/Scripting/NativeExecution/NativeLinuxCommands.Tests.ps1

```diff
@@ -1,39 +1,57 @@
-if ( $IsWindows ) {
-    $PesterSkipOrPending = @{ Skip = $true }
-}
-else {
-    $PesterSkipOrPending = @{}
-}
 Describe "NativeLinuxCommands" -tags "CI" {
+    BeforeAll {
+        $originalDefaultParams = $PSDefaultParameterValues.Clone()
+        $PSDefaultParameterValues["It:Skip"] = $IsWindows
+        $originalPath = $env:PATH
+        $env:PATH += [IO.Path]::PathSeparator + $TestDrive
+    }
+
+    AfterAll {
+        $global:PSDefaultParameterValues = $originalDefaultParams
+        $env:PATH = $originalPath
+    }
+
     It "Should return a type of 'string' for hostname cmdlet" {
```

Comment:
Now we skip the test - why?

Reply:
This test was created a long time ago.  I think it's actually not useful and is implicitly covered by tests that follow.  I'll remove it.

Reply:
Closed.